packages/coordinator/src/services/validation/bundle-validator.ts

```diff
@@ -453,13 +453,8 @@ function checkPrimaryCarePrescriptionResources(
 }
 
 function checkSecondaryCarePrescriptionResources(
-  practitionerRole: fhir.PractitionerRole,
   organization: fhir.Organization,
 ): fhir.OperationOutcomeIssue {
-  if (!practitionerRole.healthcareService) {
-    return errors.missingRequiredField("practitionerRole.healthcareService")
-  }
-
   if (organization.partOf) {
```

Comment:
this should be allowed for secondary care

Reply:
What should be allowed sorry? We are now allowing healthcareService resources for Secondary Care prescriptions, or are we saying they should be allowed to include the `organization.partOf` to replace the healthcareService resource (I think I just answered my own question)